app_test.py

```diff
@@ -27,6 +27,11 @@ def test_index(self):
         response = self.client.get('/', 'http://pytest.local')
         self.assertEqual(response.status_code, 200)
 
+    def test_help(self):
+        ''' Check the help path responds with 200 OK '''
+        response = self.client.get('/help', 'http://pytest.local/help')
```

Comment:
it's not strictly necessary to include this: `'http://pytest.local/help'`, it defaults to the main url, I'm not sure why I included it myself in the top, but it's totally optional :D